sanic_ext/extensions/openapi/types.py

```diff
@@ -65,6 +65,15 @@ class Schema(Definition):
 
     @staticmethod
     def make(value, **kwargs):
+        _type = type(value)
+        if (
+            issubclass(_type, t._GenericAlias)
+            and len(value.__args__) == 2
+            and type(None) in value.__args__
+        ):
+            value = value.__args__[0]
```

Comment:
Since we are going to arbitrarily pick the first element as the value, we might as well pick the second element arbitrarily to check if it's type(None), right?
```suggestion
            and value.__args__[1] is type(None)
        ):
            value = value.__args__[0]
```

Reply:
Or we could not pick the value non-arbitrarily, then it'll also work if someone decides to do `Union[None, str]`.

Reply:
> Since we are going to arbitrarily pick the first element as the value, we might as well pick the second element arbitrarily to check if it's type(None), right?

That is this:

```
type(None) in value.__args__
```